releasenotes/notes/parameter_conflict_in_transpile-fc67d76288b480c4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    The function ```iqiskit.compiler.transpile``` now raises an error when conlicting parameters are passed, instead of ignoring them silently. 
```

Comment:
Can you update this note for the deprecation too? looks good otherwise

Reply:
reno added in bab0cfaeb